packages/charts/react-charting/src/utilities/utilities.ts

```diff
@@ -408,10 +415,12 @@ export function createNumericYAxis(
   // maxOfYVal coming from only area chart and Grouped vertical bar chart(Calculation done at base file)
   const tempVal = maxOfYVal || yMinMaxValues.endValue;
   const finalYmax = tempVal > yMaxValue ? tempVal : yMaxValue!;
-  const finalYmin = yMinMaxValues.startValue < yMinValue ? 0 : yMinValue!;
+  const finalYmin = supportNegativeValuesForYAxis
+    ? Math.min(yMinMaxValues.startValue, yMinValue || 0)
+    : Math.max(Math.min(yMinMaxValues.startValue, yMinValue || 0), 0);
   const domainValues = prepareDatapoints(finalYmax, finalYmin, yAxisTickCount, isIntegralDataset);
   const yAxisScale = d3ScaleLinear()
-    .domain([finalYmin, domainValues[domainValues.length - 1]])
+    .domain([domainValues[0], domainValues[domainValues.length - 1]])
```

Comment:
>.domain([domainValues[0],  [](http://example.com/codeflow?start=4&length=26)

this change looks unnecessary

Reply:
yes, just to make code more understandable, made this change. will revert.

Reply:
domainValues[0] can be less than finalYmin ( for negative y value cases )